phys2bids/bids.py

```diff
@@ -170,3 +172,58 @@ def use_heuristic(heur_file, sub, ses, filename, outdir, record_label=''):
     heurpath = os.path.join(fldr, f'{name}physio')
 
     return heurpath
+
+
+def participants_file(indir, outdir, yml, sub):
+    """[summary]
+
+    Args:
+        indir ([type]): [description]
+        outdir ([type]): [description]
+        yml ([type]): [description]
+        sub ([type]): [description]
+    """
+
+    file_path = os.path.join(outdir, 'participants.tsv')
+    if not os.path.exists(file_path):
+        # Read yaml info if file exists
+        if os.path.exists(os.path.join(indir, yml)):
+            with open(os.path.join(indir, yml)) as f:
+                yaml_data = yaml.load(f, Loader=yaml.FullLoader)
+            p_id = yaml_data['participant']['participant_id']
```

Comment:
I think that in bids standard ``participant_id`` and sub are always the same therefore ``p_id=sub`` should be constant since the name of the subject has to be entered in the console argument -sub

Reply:
In my opinion ```p_id=sub`` should be out of this condition

Reply:
Fair enough. So should we use `sub` instead of the yaml data?

Reply:
Yeah that is what i was suggesting, since -sub has to be introduced for bidsdifying data